src/gms/aggregate_branch_lists.py

```diff
@@ -1,57 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
This file was deleted, but is being used/imported by `/tools/copy_test_case_folders.py`.

Reply:
replaced by a newer seperate edition that pre-existed in the src directory. Hopefully the pointers are correct now.

Reply:
added possible deprecation warning